config/crd/bases/kuadrant.io_managedzones.yaml

```diff
@@ -61,6 +61,7 @@ spec:
                 type: string
               domainName:
                 description: Domain name of this ManagedZone
+                pattern: ^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$
```

Comment:
Just noting that I've validated this. Alongside the ocm walkthrough I faced the following error after copying the file directly 
`{"error": "InvalidInput: 1 validation error detected: Value 'IDVALUE # this sets up your default managed zone' at 'resourceId' failed to satisfy constraint: Member must have length less than or equal to 32\n\tstatus code: 400, request id: a52d2233-67a4-4c6a-a3ba-610312cef431"}